rotkehlchen/chain/evm/decoding/morpho/decoder.py

```diff
@@ -258,14 +258,15 @@ def _decode_reward_claim(self, context: DecoderContext) -> DecodingOutput:
         if context.tx_log.topics[0] != CLAIM_TOPIC:
             return DEFAULT_DECODING_OUTPUT
 
+        claimed_asset = self.base.get_or_create_evm_asset(
+            address=bytes_to_address(context.tx_log.topics[2]),
+        )
         for event in context.decoded_events:
             if (
-                event.event_type == HistoryEventType.RECEIVE and
-                event.event_subtype == HistoryEventSubType.NONE and
-                event.address == self.rewards_distributor and
-                event.asset == self.base.get_or_create_evm_asset(
-                    address=bytes_to_address(context.tx_log.topics[2]),
-                )
+                    event.event_type == HistoryEventType.RECEIVE and
+                    event.event_subtype == HistoryEventSubType.NONE and
+                    event.address == context.tx_log.address and
```

Comment:
Before we had `event.address == self.rewards_distributor`. Was it wrong?

Reply:
now there is multiple distributors. And this event is always fired by the distributor.